### PR TITLE
Selenium: Use Options, as Remote::Capabilities is deprecated

### PR DESCRIPTION
**Pull request by liangxia**
To address the issue
```
2023-02-09 04:36:39 WARN Selenium [DEPRECATION] [:caps_browsers] Remote::Capabilities.chrome is deprecated. Use Options.chrome instead.
```

/cc @yanpzhan @yapei @XiyunZhao
/cc @jhou1 @pruan-rht @JianLi-RH @dis016 

**Please note: this is in the devel branch.**
We can test it after
1. The current test in https://github.com/openshift/release/pull/36116 finished
2. This PR is reviewed and merge.

### CODE REVIEW

File: lib/webauto/web4cucumber.rb [hunk 1]
```diff
@@ -74,15 +74,15 @@ def started?
...
+      chrome_opts = Selenium::WebDriver::Options.chrome
       safari_caps = Selenium::WebDriver::Remote::Capabilities.safari()
       chrome_switches = []
       if ENV.has_key?("http_proxy") || @http_proxy
         # get rid of the heading "http://" that breaks the profile
         proxy_raw = (@http_proxy || ENV["http_proxy"]).sub(/(\/)+$/, '')
         proxy = proxy_raw.sub(%r{^.+?://}, "")
         proxy_bypass = "localhost,127.0.0.1"
-        firefox_profile.proxy = chrome_caps.proxy = safari_caps.proxy = Selenium::WebDriver::Proxy.new({:http => proxy.sub(%r{^.+?@}, ""), :ssl => proxy.sub(%r{^.+?@}, "")})
+        firefox_profile.proxy = chrome_opts.proxy = safari_caps.proxy = Selenium::WebDriver::Proxy.new({:http => proxy.sub(%r{^.+?@}, ""), :ssl => proxy.sub(%r{^.+?@}, "")})
```
Comment (yapei) [inline lib/webauto/web4cucumber.rb:85]: it looks like although we set `chrome_opts.proxy` here but it is never used ~ (same to chrome_caps)

File: lib/webauto/web4cucumber.rb [hunk 2]
```diff
@@ -74,15 +74,15 @@ def started?
     def browser
       return @browser if @browser && @browser.exists?
       firefox_profile = Selenium::WebDriver::Firefox::Profile.new
-      chrome_caps = Selenium::WebDriver::Remote::Capabilities.chrome()
+      chrome_opts = Selenium::WebDriver::Options.chrome
```
Comment (yapei) [inline lib/webauto/web4cucumber.rb:77]: Can we also remove `Selenium::WebDriver::Remote::Capabilities.safari()` to resolve following warning? 

```
02-10 16:54:49.791  2023-02-10 08:54:49 WARN Selenium [DEPRECATION] [:caps_browsers] Remote::Capabilities.safari is deprecated. Use Options.safari instead.
```

Comment (liangxia) [reply lib/webauto/web4cucumber.rb:77]: We can do that in a follow up PR after we verified the changes work in both Jenkins CI and Prow CI.

File: lib/webauto/web4cucumber.rb (hunk 1 above)
Comment (liangxia) [reply lib/webauto/web4cucumber.rb:85]: I haven't do a deep dive into the proxy part, so I just leave it as it is now. Perhaps we can do the cleanup in a follow up PR. WDYT ?

**Comment by liangxia** [comment]
To speed up the code change/review/test process, I think we can skip the review there as devel branch is used for test purpose.
I'll just get the code changes into devel branch, and test it in Jenkins CI and Prow CI.  Once the changes looks good, will send out the PR in master branch for review.

**Comment by liangxia** [comment]
Replaced by https://github.com/openshift/verification-tests/pull/3216

**Pull request closed without merging**